sdk/monitor/arm-monitor/_meta.json

```diff
@@ -1,8 +1,8 @@
 {
-  "commit": "dcf548aea9f776d166e8c53f8ecb8eff9beef2a5",
+  "commit": "6fc313ed7a6e926d4409d4ddb435f6532922652f",
```

Comment:
@qiaozha I plan to relese

Reply:
Offline confirmed and our PM has sent the confirmation with service team and we take this as silent agreement.